test_pytapo.py

```diff
@@ -479,6 +479,11 @@ def test_moveMotorStep():
     assert result2["error_code"] == 0
 
 
+def test_calibrateMotor():
+    tapo = Tapo(host, user, password)
+    tapo.calibrateMotor()
```

Comment:
We should test for something here, maybe move the camera via ptz before/after and compare values.. or just compare the result string if it didnt throw an error

Reply:
How though?

Also the tests aren't meant to test whether the camera works, but whether the library works. If it doesn't return an error, then it's fine.

TBH you shouldn't be doing tests with the camera at all, the camera should be mocked.

Reply:
Like this:

assert result["error_code"] == 0

Its integration vs unit tests. I want to actually test the camera to verify functionality of the whole flow.

For example, in the future change in firmware could render some functionality not working. Unit tests would not catch this.

Code base is also not so complex to need unit tests (with mock), it's basically really simple lib. What we need is integration tests that fully test the whole flow.